lib/ex_doc/formatter/html.ex

```diff
@@ -146,18 +146,46 @@ defmodule ExDoc.Formatter.HTML do
     |> ExDoc.DocAST.highlight(language, opts)
   end
 
-  defp output_setup(build, config) do
+  def setup_output(config, cleanup, create, root \\ nil) do
```

Comment:
It is currently that way because in the EPUB example, `normalize_config` appends something to the path, so that `config.output` is not the root directory.

https://github.com/elixir-lang/ex_doc/blob/b66861bb60173b056022e95662cf72275eba18a8/lib/ex_doc/formatter/epub.ex#L13

But thinking a bit better, I think it is possible to just call `setup_output` before `normalize_config`